sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -604,7 +604,7 @@ class Analyzer(
 
     def apply(plan: LogicalPlan): LogicalPlan = plan resolveOperators {
       case i @ InsertIntoTable(u: UnresolvedRelation, parts, child, _, _) if child.resolved =>
-        i.copy(table = EliminateSubqueryAliases(lookupTableFromCatalog(u)))
+        i.copy(table = resolveRelation(EliminateSubqueryAliases(lookupTableFromCatalog(u))))
```

Comment:
why do we need to resolve the child of view? Once we see the pattern `Insert(View, ...)` we will throw exception, we don't care about whether the child of view is resolved or not.